build-vm

```diff
@@ -566,6 +566,14 @@ vm_detect_2nd_stage() {
 	    echo "resolved swap device $VM_SWAP to $i"
 	    VM_SWAP=$i
 	fi
+        # for zvm, there is just a 4-digig hex number
+            if (( 16#$VM_SWAP )); then
+                if test ${#VM_SWAP} -eq 4; then
```

Comment:
this looks not like valid bash syntax.

Also some quotes to protect against invalid data seems to be missing

Reply:
hmm ... I actuallyl thought it might be nice to redirect the output to /dev/null, however the code works. First test is, if $VM_SWAP is a hex number, second is, if it has 4 digits:
> (( 16#1f )); echo $?
0
> (( 16#1g )); echo $?
bash: ((: 16#1g: value too great for base (error token is "16#1g")
1

> VM_SWAP=12345
> echo ${#VM_SWAP}
5